Jenkinsfile

```diff
@@ -59,7 +64,16 @@ pipeline {
                         env.testProvider = "nfs"
                     } else if (body.contains("[test yugabytedb]")) {
                         env.testProvider = "yugabytedb"
+                    } else if (body.contains("[test]")) {
+                        env.shouldBuild = "true"
+                    } else if (!changed_files.contains(".go") && (changed_files.contains(".md") || changed_files.contains(".yaml") || changed_files.contains(".txt"))) {
```

Comment:
Using a whitelist may be too aggressive to skip the CI (e.g. if you have a makefile in the same PR as an .md file). Since maintainers already need to review the CI before merging a PR we can give this a try, then adjust if it is too permissive.

Reply:
Let's start by solving the 90% first :)